tutorial/Part 1 - Train on custom dataset.md

```diff
@@ -0,0 +1,174 @@
+# Part 1 - Train on custom dataset
```

Comment:
We can review this file once we have a clear idea of what the objects/classes would be.